From: =?utf-8?q?Tim_R=C3=BChsen?= <tim.ruehsen@gmx.de>
Date: Sun, 26 May 2019 17:59:44 +0200
Subject: Add PSL_API to libpsl-sections.txt

(cherry picked from commit ef352f8923ada2ec766f8bfdefc86ffd8402af09)
---
 docs/libpsl/libpsl-sections.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/docs/libpsl/libpsl-sections.txt b/docs/libpsl/libpsl-sections.txt
index c858051..ba7cf06 100644
--- a/docs/libpsl/libpsl-sections.txt
+++ b/docs/libpsl/libpsl-sections.txt
@@ -1,6 +1,7 @@
 <SECTION>
 <FILE>libpsl</FILE>
 <TITLE>Public Suffix List functions</TITLE>
+PSL_API
 PSL_VERSION
 PSL_VERSION_MAJOR
 PSL_VERSION_MINOR
